circle.yml

```diff
@@ -419,16 +436,6 @@ jobs:
               echo "Skipping Record size step"
               # Skipping due to https://github.com/mapbox/mapbox-gl-native/issues/15751
               #platform/ios/scripts/metrics.sh
-      - run:
-          name: Trigger metrics
-          command: |
-            if [[ $CIRCLE_BRANCH == master ]]; then
-              if [ -n "${MOBILE_METRICS_TOKEN}" ]; then
-                curl -u ${CIRCLE_CI_TOKEN}: -X POST --header "Content-Type: application/json" --data '{"parameters": {"run_ios_maps_benchmark": true, "ci_ref" : $CIRCLE_BUILD_NUM }, "branch": "master"}' https://circleci.com/api/v2/project/github/mapbox/mobile-metrics/pipeline
-              else
-                echo "MOBILE_METRICS_TOKEN not provided"
-              fi
-            fi
```

Comment:
Should we break this out into a separate script? Not sure how much we want to do in the config.yml itself.